fe/fe-core/src/main/java/com/starrocks/catalog/Table.java

```diff
@@ -765,6 +765,10 @@ public Map<String, String> getProperties() {
         throw new NotImplementedException();
     }
 
+    public Map<String, String> getUniqueProperties() {
```

Comment:
It seems that `getUniqueProperties` is an API to get properties that are only available on tables of a specific type? Can you give some comments for the difference between getProperties and getUniqueProperties.

Reply:
ok, and move `getUniqueProperties` api from `Table` to `OlapTable` because this is only used in `OlapTable` and its subclasses.